flink-dist/pom.xml

```diff
@@ -113,8 +113,13 @@ under the License.
 			<artifactId>flink-metrics-jmx</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+        
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-mesos_2.10</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
We always build yarn. We use the `include-yarn-tests` profile to include/exclude yarn tests. The `include-yarn` profile, on the other hand, it to exclude yarn for the Haodop 1 version of Flink.